netdeploy/networkTemplate.go

```diff
@@ -242,6 +276,17 @@ func countRelayNodes(nodeCfgs []remote.NodeConfigGoal) (relayCount int) {
 	return
 }
 
+func decodeJSONOverride(override string, cfg *config.Local) error {
+	if override != "" {
+		reader := strings.NewReader(override)
+		dec := codec.NewDecoder(reader, protocol.JSONHandle)
```

Comment:
Oh cool. I always assumed the main reason for go-codec was because the generated encoders/decoders are going to be significantly faster than encoding/decoding via reflection.

I'll switch this to using the built-in feature.

Reply:
done.